easybuild/easyconfigs/i/iomkl/iomkl-2017.09.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2017.09'
```

Comment:
@jhein32 didn't we agree on also making this `2017b`?

Reply:
Not that I know off (I might have overread something though).  You feel strongly about this?

Reply:
@boegel If you want, I can do this first thing tomorrow morning.  Need to run for today.

Reply:
Since this `iomkl` definition is really close to `intel/2017b` and just replaces the `impi` in there with the `OpenMPI` from `foss/2017b`, I think it makes total sense to go with `2017b` as a version here.

We've done this in the past for `gmpolf` and `gmvolf` too, and I'm starting to consider to also make `iomkl` part of the set of common toolchains...

Reply:
@boegel Done, it is iomkl 2017b now.  Haven't done any test beyond the travis, see below.